templates/includes/image_gallery.html

```diff
@@ -127,7 +127,8 @@ <h2>{{ item.image.get_translation_heading_display }}</h2>
     {% endfor %}
 {% endwith %}
 <button class="transcription__close hidden" id="closeButton" type="button">
-    Close
+    Close <span class="sr-only">images
```

Comment:
For consistency with the 'View images and transcripts' button lets keep this text visible for all users by removing the `sr-only` class - then it's really clear for all users what the button is closing.